pkg/api/addons.go

```diff
@@ -322,15 +322,11 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 		Containers: []KubernetesContainerSpec{
 			{
 				Name:  common.AzureNetworkPolicyAddonName,
-				Image: "mcr.microsoft.com/containernetworking/azure-npm:v1.0.29",
+				Image: "mcr.microsoft.com/containernetworking/azure-npm:v1.0.30",
```

Comment:
After this line we will add the following 2 container-specific configuration values to enable memory resource limits and requests:

```
MemoryRequests: "20Mi",
MemoryLimits:   "2Gi",
```

Reply:
added